Module.php

```diff
@@ -144,8 +144,9 @@ public function bootstrap($app)
         $app->on(Application::EVENT_BEFORE_REQUEST, function () use ($app) {
             $app->getView()->on(View::EVENT_END_BODY, [$this, 'renderToolbar']);
         });
-
-        $app->getUrlManager()->addRules([
+        $urlManager = $app->getUrlManager();
+        $urlManager->ruleConfig['class'] = 'yii\web\UrlRule';
```

Comment:
That's when app's URLManager is configured to use non-default rule by default for whatever reason and thus debug module can't function because of such default.